javascript/javascript1/week3/homeworkweek03.js

```diff
@@ -0,0 +1,113 @@
+const names = ['Peter', 'Ahmad', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala'];
+const nameToRemove = 'Ahmad';
+
+// Write some code here
+names.splice(1,1);
+// Code done
+
+console.log(names); 
+// Exercise 2 
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  function notThisFunctionName(travelInformation){
+          const a = travelInformation.destinationDistance / travelInformation.speed;
```

Comment:
Variable naming&  also function naming conventions , hour/minute - is a general naming, be very specific.
ex: travelInHour etc...